doctor-module/src/main/java/com/khadri/spring/core/doctror/processor/InPatientProcessor.java

```diff
@@ -0,0 +1,9 @@
+package com.khadri.spring.core.doctror.processor;
+
+public class InPatientProcessor {
```

Comment:
why you need to create , you just add dependency of module so this class is available for auto wiring.